### ISSUE TITLE
Fails to install `dependecy-git` when installing `package-git` with `--devel`

### ISSUE THREAD

**Issue by lucc** [rmarquis/pacaur#396]
I found this with the package `rolo` and `rolo-git` which depend on `libvc` and `lbvc-git` respectively.

Normally I upgrade and install with `pacaur -Syu --devel --needed`. This works for `rolo` but not for `rolo-git`. The shortest option list which reproduces the failure is `pacaur -S --devel rolo-git` (I know `--devel` doesn't make much sense here but all other subsets of options that include `--devel` also fail).

The output:
```
~ $ pacaur -S --devel rolo-git
:: Package(s) rolo-git not found in repositories, trying AUR...
:: resolving dependencies...
:: looking for inter-conflicts...

AUR Packages  (1): rolo-git-latest  

:: Proceed with installation? [Y/n] 

:: Retrieving package(s)...
Already up-to-date.
:: View rolo-git PKGBUILD? [Y/n] n
:: Checking rolo-git integrity...
==> Making package: rolo-git 1-1 (Wed Jan 13 10:01:02 CET 2016)
==> Retrieving sources...
  -> Updating rolo git repo...
Fetching origin
==> Validating source files with md5sums...
    rolo ... Skipped
:: Building rolo-git package(s)...
==> Making package: rolo-git 1-1 (Wed Jan 13 10:01:04 CET 2016)
==> Checking runtime dependencies...
==> Installing missing dependencies...
error: target not found: libvc-git
==> ERROR: 'pacman' failed to install missing dependencies.
:: Installing rolo-git package(s)...
:: failed to build rolo-git package(s)
~ $
```
Config files (seem to be irrelevant):
`~/.config/pacaur/config`:
```
clonedir=~/.cache/pacaur/src
```
`~/.config/cower/config`:
```
Color = auto
TargetDir = ~/build
MaxThreads = 2
```

**Comment by rmarquis** [comment]
I cannot reproduce. In fact, your command produces the following output on my machine:

```
$ pacaur -S --devel rolo-git
:: Package(s) rolo-git not found in repositories, trying AUR...
:: resolving dependencies...
:: looking for inter-conflicts...
:: rolo-git latest revision -- fetching

AUR Packages  (2):

Name           Old Version  New Version

aur/libvc-git  1-1          latest
aur/rolo-git   1-1          latest

:: Proceed with installation? [Y/n]
```

Please check that you're using the latest stable version of pacaur, cower and expac, and that you don't use any 3rd party repository that might interfere with AUR packages: the package ```libvc-git``` isn't detected as an AUR package, but as a pre-compiled package for some reason.

**Comment by lucc** [comment]
I'm using the following (package) versions:
- pacaur 4.4.3-1
- cower 14-2
- expac 4-3

Are there any config files that I might have overlooked?

**Comment by rmarquis** [comment]
Your packages are up-to-date. Are you sure you don't use any additional third party repository? Check your ```pacman.conf```. At second sight, I'd also say ```MaxThreads``` in cower config file could be suspicious, but it shouldn't have any real effect on cower (only speed). Any reason you're using this setting? Also, do you use any wrapper between pacaur and pacman, such as powerpill?
Lastly, please provide a debug output with ```bash -x pacaur -S --devel rolo-git```. This issue is weird.

**Comment by lucc** [comment]
Here is the diff of my `pacman.conf` (generated with `acat /var/cache/pacman/pkg/pacman-4.2.1-4-x86_64.pkg.tar.xz etc/pacman.conf  2>/dev/null| diff - /etc/pacman.conf`):
```diff
29c29,35
< #NoExtract   =
---
> 
> # Do not install unneeded language files.  Warning:  This was making trouble
> # after reinstalling base and base-devel.  I had to uncomment this and rerun
> # locale-gen.
> #NoExtract   = usr/share/locale/*
> #NoExtract   = !usr/share/locale/de*
> #NoExtract   = !usr/share/locale/en*
33c39
< #Color
---
> Color
76a83,86
> #[haskell-core]
> #Server = http://xsounds.org/~haskell/core/$arch
> #Server = http://www.kiwilight.com/haskell/core/$arch
> 
92,93c102,111
< #[multilib]
< #Include = /etc/pacman.d/mirrorlist
---
> [multilib]
> Include = /etc/pacman.d/mirrorlist
> 
> #[archlinuxfr]
> #SigLevel = Never
> #Server = http://repo.archlinux.fr/$arch
> 
> #[kps]
> #SigLevel = PackageOptional
> #Server = http://kps.bplaced.net/repo/$arch
```
So that's just some comments, the color option and the multilib repo.

I moved my cower config file away and ran `bash -x pacaur -S --devel rolo-git >pacaur.log 2>&1` (blindly pressing ENTER and n) to create [the log file](https://gist.github.com/lucc/962fe13d6fb50606ce08).

**Comment by rmarquis** [comment]
Thanks, I located the issue. This has been introduced in https://github.com/rmarquis/pacaur/commit/ff07757fb8a2d00ca33ce224405a19c2db4dae44 a few months ago. Installing packages with ```-Su --devel package``` is fairly uncommon (but still valid), so that bug has been undetected until now.

**Comment by rmarquis** [comment]
Fixed in 35fbbbc2d8b4eef5cd3da2a59c8395488d298a01. New update on the way!

**Comment by lucc** [comment]
thanks
